CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Hygieia Contribution Guidelines
+
+Thank you for your interest in contributing to Hygieia! It is because of developers like you that this project is able to succeed.
+
+In order to ensure the highest standards of security and code quality, we have laid out some guidelines that should be followed when contributing to Hygieia.
+
+## General Pull Request (PR) Guidelines
+
+* PR's should fix one issue or add/update one feature. If you would like to update multiple things, please separate into individual PRs so that they can be adequately reviewed.
+* PR titles should be succint and descriptive. The "description" field should specify which issue/feature the PR relates to, give an overview of changes made, and highlight any specific lines or files that may need extra review.
+* If applicable, please include screenshots or videos of new features to assist in review.
+
+## Testing and Linting
+* Please run ```ng lint``` and address any errors before requesting review.
+* We use [Karma](https://karma-runner.github.io/latest/index.html) for testing. Before opening your PR, please add unit tests for any new features and update any existing ones necessary for the test suite to pass.
+* Testing thresholds must be met as follows:
+  * Branch Coverage: 80%
+  * Line Coverage: 80%
+  * Statement Coverage: 80%
+  * Function Coverage: 80%
```

Comment:
Possibly collapse this since every benchmark is 80%. Maybe something like:
`Hygieia requires wholistic code coverage above the 80% mark, including Branch, Line, Statement, and Function coverage`

Once again, simple preference.